deployment/jenkins/azureutil.groovy

```diff
@@ -66,36 +66,32 @@ def prepareEnv(String targetEnv) {
 
 def deployFunction() {
     sh """
-        # Storage connection for images 
+        # Storage connection for images
         storage_name=\$(az storage account list -g ${config.COMMON_GROUP} --query [2].name | tr -d '"')
         storage_conn_str=\$(az storage account show-connection-string -g ${config.COMMON_GROUP} -n \${storage_name} --query connectionString | tr -d '"')
-        
+
         function_id=\$(az functionapp list -g ${config.COMMON_GROUP} --query [0].id | tr -d '"')
         az functionapp config appsettings set --ids \${function_id} --settings STORAGE_CONNECTION_STRING=\${storage_conn_str}
         az functionapp deployment source sync --ids \${function_id}
     """
 }
 
-def deployWebApp(String resGroup) {
+def deployWebApp(String resGroup, String appName, String dockerFilePath) {
+    azureWebAppPublish appName: appName, azureCredentialsId: 'azure-sp', dockerFilePath: dockerFilePath, dockerImageName: '', dockerImageTag: '', dockerRegistryEndpoint: [credentialsId: 'acr', url: "https://${ACR_NAME}.azurecr.io"], filePath: '', publishType: 'docker', resourceGroup: resGroup, slotName: '', sourceDirectory: '', targetDirectory: ''
```

Comment:
As we talked offline, this line doesn't support multiple environments.
Any update on the fix?

Reply:
Sorry for late as I'm engaged in other projects. I'll work on a new patch in next a few days.

Reply:
Cool. Thanks for the update . 😃

Reply:
Patch updated.

Note that since I changed jenkins master/slave docker images, we also need to build and  publish them to docker hub once this get merged.